rudder-lang/src/bin/rudderc.rs

```diff
@@ -86,12 +72,19 @@ fn main() {
     info!("I/O context: {}", ctx);
 
     // Actual action
-    // TODO : check compiler github ; update geerators / translation
-    // json -> rl (/ cf / dsc ?)
-    // rl -> json / cf / dsc
+    // read = rl -> json
+    // migrate = cf -> rl
+    // compile = rl -> cf / dsc
+    // generate = json -> rl / cf / dsc
+    // TODO make the right calls
     let result = match action {
         Action::Compile => compile_file(&ctx, true),
-        Action::Translate => generate(&ctx),
```

Comment:
Where does the generate call go ?

Reply:
disappears in next PR, unused since actions are not Translate and Compile anymore